website/files/models/base.py

```diff
@@ -398,9 +399,10 @@ def copy_under(self, destination_parent, name=None):
 
     def move_under(self, destination_parent, name=None):
         self.name = name or self.name
+        if self.parent.node != destination_parent.node:
+            project_signals.file_moved_node.send(self, destination_node=destination_parent.node)
```

Comment:
I see that, but other listeners might find it useful. Again, it's minor, and doesn't need to be done at this point.